### PR TITLE
Tidy up a few typos in test code

### PR DESCRIPTION
**Pull request by dwlsalmeida**
Signed-off-by: Daniel W. S. Almeida <dwlsalmeida@gmail.com>

### CODE REVIEW

File: test/export_cube_scaledXYZ.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def test_scaling_xyz_by_2(self):
 
 if __name__ == '__main__':
     suite_1 = unittest.defaultTestLoader.\
-            loadTestsFromTestCase(TestExportCubeScaledX)
```
Comment (GENIVI-Ramses) [inline test/export_cube_scaledXYZ.py]: is already fixed on master

File: test/export_cube_scaledY.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def test_scaling_y_by_2(self):
 
 if __name__ == '__main__':
     suite_1 = unittest.defaultTestLoader.\
-            loadTestsFromTestCase(TestExportCubeScaledX)
```
Comment (GENIVI-Ramses) [inline test/export_cube_scaledY.py]: same

File: test/export_cube_scaledZ.py [hunk 3]
```diff
@@ -70,7 +70,7 @@ def test_scaling_z_by_2(self):
 
 if __name__ == '__main__':
     suite_1 = unittest.defaultTestLoader.\
-            loadTestsFromTestCase(TestExportCubeScaledX)
```
Comment (GENIVI-Ramses) [inline test/export_cube_scaledZ.py]: same

File: test/run_all_tests.py [hunk 4]
```diff
@@ -119,20 +119,6 @@ def main():
...
-                'expected_image': os.path.join(current_path, 'expected_results/cube_moved.png'),
-                'expected_output_files': 6,
-            },
-        'export_rotatedX_cube' :
-            {
-                'script'        : os.path.join(current_path, 'export_cube.py'),
-                'test_scene'    : os.path.join(current_path, 'test_scenes/cube_rotatedX.blend'),
-                'expected_image': os.path.join(current_path, 'expected_results/cube_rotatedX.png'),
-                'expected_output_files': 6,
-            }
```
Comment (GENIVI-Ramses) [inline test/run_all_tests.py]: why remove these tests?

Comment (dwlsalmeida) [reply test/run_all_tests.py]: These are to be replaced by more specific tests. For instance, tests for rotation are comprised of rotations on different axis, by different angles and in different rotation ordering. You can check this in #6

**Comment by dwlsalmeida** [comment]
3f5124a Rebased on top of the last master

**Pull request merged**